sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/models/QueueServiceProperties.java

```diff
@@ -22,7 +22,7 @@
      * Azure Analytics Logging settings
      */
     @JsonProperty(value = "Logging")
-    private QueueAnalyticsLogging logging;
+    private QueueAnalyticsLogging analyticsLogging;
```

Comment:
Did you rename this manually? This needs a swagger change as well.

Reply:
added rename rule to swagger transformation file.